proposals.go

```diff
@@ -0,0 +1,93 @@
+package dcrlibwallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	proposalsEndpoint = "https://proposals.decred.org/api/v1/proposals/vetted"
+	policyEndpoint    = "https://proposals.decred.org/api/v1/policy"
+)
+
+func (mw *MultiWallet) createHTTPClient() {
```

Comment:
I do not think any of the functions in this file should depend on a multiwallet instance, it should standalone so it can be invoked and managed separately.